pkg/api/types.go

```diff
@@ -887,6 +887,11 @@ type QuobyteVolumeSource struct {
 	// Default is no group
 	// +optional
 	Group string
+
+	// Name of Quobyte Tenant that the Volume is part of
+	// Default is DEFAULT
```

Comment:
putting this in a pod spec allows a pod to specify any tenant... is that expected, or is this intended for persistent volume sources only?